packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -34,16 +39,16 @@ export const Select: FC<ISelectProps> = forwardRef<
     disabled = false,
     children,
     ariaLabel,
+    variant = variants[0],
```

Comment:
```suggestion
    variant = "form",
```

Reply:
Wait what why do you suggest this? It would mean that you have to change it multiple places if we decide to change the variant name. Having this default to the first option within the variants makes sense I think?